### PR TITLE
Translate Ruby 2.4.0-preview1 release post (pl)

### PR DESCRIPTION
**Pull request by Quintasan**

### CODE REVIEW

File: pl/news/_posts/2016-06-20-ruby-2-4-0-preview1-released.md [hunk 1]
```diff
@@ -0,0 +1,113 @@
...
+Metody `String/Symbol#upcase/downcase/swapcase/capitalize(!)` od teraz
+potrafią zmieniać kapitalizację znaków Unicode.
+
+## Poprawki wypływające na wydajność
+
+Ruby 2.4 zawiera poniższe usprawnienia pozytywnie wypływające na wydajność:
+
+### [Array#max, Array#min](https://bugs.ruby-lang.org/issues/12172)
+
+`[x, y].max` and `[x, y].min` zostały zoptymalizowane tak,
```
Comment (stomar) [inline pl/news/_posts/2016-06-20-ruby-2-4-0-preview1-released.md]: "and"

File: pl/news/_posts/2016-06-20-ruby-2-4-0-preview1-released.md [hunk 2]
```diff
@@ -0,0 +1,113 @@
...
+zawierają listę zmian.
+
+Od wydania Rubiego 2.3 zmieniło się
+[1140 plików, dodano 33126 linii kodu(+) a usunięto 50993(-)](https://github.com/ruby/ruby/compare/v2_3_0...v2_4_0_preview1)
+
+## Pobierz
+
+* <https://cache.ruby-lang.org/pub/ruby/2.4/ruby-2.4.0-preview1.tar.bz2>
+
+  * Rozmiar:   14108114 bajtów
```
Comment (stomar) [inline pl/news/_posts/2016-06-20-ruby-2-4-0-preview1-released.md]: Download section should be untranslated (size, bytes, ...)

**Comment by stomar** [comment]
Please squash your fixes into the first commit and force-push your branch.

**Comment by JuanitoFatas** [comment]
Merged as afb8881. Thank you Quintasan!!! ❤️

**Pull request closed without merging**